cross/x264/Makefile

```diff
@@ -1,20 +1,52 @@
 PKG_NAME = x264
-PKG_VERS = 20141218-2245
+PKG_VERS = snapshot-20160517-2245-stable
 PKG_EXT = tar.bz2
-PKG_DIST_NAME = $(PKG_NAME)-snapshot-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = ftp://ftp.videolan.org/pub/x264/snapshots/
-PKG_DIR = $(PKG_NAME)-snapshot-$(PKG_VERS)
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://ftp.videolan.org/pub/videolan/x264/snapshots
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
 DEPENDS =
-GNU_CONFIGURE = 1
+
 HOMEPAGE = http://www.videolan.org/developers/x264.html
 COMMENT  = x264 is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC forma
 LICENSE  =
 
-CONFIGURE_ARGS = --enable-static --disable-asm --prefix=$(INSTALL_PREFIX)
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --enable-static --prefix=$(INSTALL_PREFIX) --enable-pic --cross-prefix=$(TC_PATH)$(TC_PREFIX)
```

Comment:
Is `--enable-static` useful here? Seeing as we include the shared libs in the resulting SPK anyway.